org.eclipse.acute/src/org/eclipse/acute/AcutePreferencePage.java

```diff
@@ -66,16 +71,44 @@ private boolean isPageValid() {
 		}
 		File dotnetCommand = new File(explicitDotnetPathText.getText());
 		if (!dotnetCommand.exists() || !dotnetCommand.isFile()) {
-			setErrorMessage("Input a valid path to `dotnet` command");
+			setErrorMessage("Input a valid path to `dotnet` command file path");
 			return false;
 		} else if (!dotnetCommand.canExecute()) {
 			setErrorMessage("Inputted command is not executable");
 			return false;
 		}
+		String version = getDotnetVersion();
+		if (version.isEmpty() || !version.matches("\\d+\\.\\d+\\.\\d+.*")) {
```

Comment:
I believe we'll need to factorize a few util methods about dotnet version, such as `DotnetVersionUtil.isValidVersion()` and `DotnetVersionUtil.getMajor(version)` and `DotnetVersionUtil.getDotnetVersion(dotnetPath)`... to remove duplication.